<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerSettingDialog</class>
 <widget class="QDialog" name="ServerSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>23</y>
     <width>120</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器IP地址：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>54</y>
     <width>120</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>gate服务端口：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>86</y>
     <width>120</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>http服务端口：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>16</y>
     <width>53</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leServerIp">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>22</y>
     <width>138</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>106.14.172.35</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leGatePort">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>53</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>8000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leHttpPort">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>85</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>18080</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
